Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -62,20 +56,26 @@ public class NetworkClient : MonoBehaviour
         /// </summary>
         public bool isLocalClient => hostServer != null;
 
+        /// <summary>
+        /// The IP address of the server that this client is connected to.
+        /// </summary>
+        [Tooltip("Network Address where client should connect to the server.")]
+        public string serverIp = "localhost";
```

Comment:
What if we put it in the transport instead.   Do we need to this in the NetworkClient?